app/pods/publications/publication/case/route.js

```diff
@@ -3,8 +3,7 @@ import { hash } from 'rsvp';
 
 export default class CaseRoute extends Route {
   async model() {
-    const parentHash = this.modelFor('publications.publication');
-    const publicationFlow = parentHash.publicationFlow;
+    const publicationFlow = this.modelFor('publications.publication');
     const _case = await publicationFlow.get('case');
```

Comment:
`case` is wat centraal staat in deze route. De hier secundaire `publicationFlow` en `contactPersons` kunnen in `afterModel`/`setupController` gezet worden.

Reply:
OK, wist niet dat ik deze ook moest refactoren. Gebeurd nu.